todo_app/app.py

```diff
@@ -1,5 +1,5 @@
 from flask import Flask, render_template, request, redirect, url_for
-from todo_app.data.session_items import get_items, add_item
+from todo_app.data.trello_items import get_items, add_item, complete_item
```

Comment:
Nice, the fact this is such a clean change in this file is a great indicator you've got a clean interface 😄